newIDE/app/src/ObjectsList/index.js

```diff
@@ -72,6 +72,8 @@ const objectTypeToDefaultName = {
   'MyDummyExtension::DummyObject': 'NewDummyObject',
   'Lighting::LightObject': 'NewLight',
   'TextInput::TextInputObject': 'NewTextInput',
+  'Scene3D::Model3DObject': 'NewModel3D',
```

Comment:
do we have restrictions on the name or can we use `New3DModel` here if this is just user-facing?
It's not consistent with the code, but I think it's more english